### ISSUE TITLE
About training parameters

### ISSUE THREAD

**Issue by biycyr** [zhreshold/mxnet-ssd#50]
I have been trying to reproduce the reported performance(71.57%) by training. 

`python train.py --gpus 0`

then

`python evaluate.py --gpus 0 --epoch 200`

However,  I only got mAP 69.79% with one GPU. I tried adding another 100 epochs and changing the learning rate and other parameters slightly. The best I can get is still only 69.98%.

Did I miss something? Does any one have the proper training parameters to achieve the reported performance?

Thanks!

**Comment by zhreshold** [comment]
I could get 70.8 before 100 epochs if use random padding in config.py
The latest updates from author have shown that extensive data augmentation is very important, which will be improved in the next version, but I can't guarantee when I'm gonna start working on it.

**Comment by biycyr** [comment]
Do you mean RandomPadders in config.py

`cfg.TRAIN.RAND_SAMPLERS = [RandCropper(min_scale=1., max_trials=1, max_sample=1),
    RandCropper(min_scale=.3, min_aspect_ratio=.5, max_aspect_ratio=2., min_overlap=.1),
    RandCropper(min_scale=.3, min_aspect_ratio=.5, max_aspect_ratio=2., min_overlap=.3),
    RandCropper(min_scale=.3, min_aspect_ratio=.5, max_aspect_ratio=2., min_overlap=.5),
    RandCropper(min_scale=.3, min_aspect_ratio=.5, max_aspect_ratio=2., min_overlap=.7),
    RandPadder(max_scale=2., min_aspect_ratio=.5, max_aspect_ratio=2., min_gt_scale=.05),
    RandPadder(max_scale=3., min_aspect_ratio=.5, max_aspect_ratio=2., min_gt_scale=.05),
    RandPadder(max_scale=4., min_aspect_ratio=.5, max_aspect_ratio=2., min_gt_scale=.05),]
`
? I thought I did use random padding since I did not change this part for training.  Did I get it wrong?

In addition, by 100 epochs, do you mean your trained the model with 100 epochs or added another 100 epochs on top of the original 200 epochs as I mentioned in previous comment?

**Comment by zhreshold** [comment]
It looks good. Training from scratch with 100 epochs is good enough, 100-200 won't do too much to the mAP. 
I think 1% difference is acceptable since it's just a random optimization. But I will try to improve it to match the latest caffe version.

**Comment by biycyr** [comment]
Good to know. thanks!
